### ISSUE TITLE
DF Module Name: Summarize issue...

### ISSUE THREAD

**Issue by duckpuppy** [flamewave000/dragonflagon-fvtt#121]
**Module**
DF Chat Enhancements v3.02

**Describe the issue**
Adding a new entry to either the Adventure Log or GM Only Log using any method inserts a new header into the journal but does not create a new entry. There are no errors in the console.

**To Reproduce**
Add a new log entry (right-click, `/log` in the chat, `/gmlog` in the chat)
Open associated journal - see new additional header for the journal type (adventure or GM), but not the actual entry.

**Expected behavior**
A new entry is added to the journal

**Screenshots (Optional)**
The screenshot below shows two `/gmlog` entries at the end of the chat log, and the corresponding headers in the GM only log. There's 3 - the original one created by the module when the journal was chosen as the GM only log, and the two created for each `/gmlog` chat command.

![Screen Shot 2021-05-27 at 2 16 38 PM](https://user-images.githubusercontent.com/19253/119876732-324be900-bef6-11eb-847f-1d9580909849.png)

**Comment by duckpuppy** [comment]
Because it might be useful, here is the source code from the above journal:

```html
<section>
<h2>GM Only Adventure Log</h2>
<hr /></section>
<section>
<h2>GM Only Adventure Log</h2>
<hr /></section>
<section>
<h2>GM Only Adventure Log</h2>
<hr /></section>
```

**Comment by flamewave000** [comment]
Yeah this looks like maybe an issue with another module modifying the journal content. That HTML is actually missing some content that is required for the logger to function. The HTML should look something like this:

```html
<section>
<h2>GM Only Adventure Log</h2>
<article class="df-adventure-log"></article>
<hr /></section>
```

The key here is the `<article class="df-adventure-log"></article>` tag which is what the code is searching for to inject the log entries into that article block.

**Comment by nat212** [comment]
I just managed to replicate this with only DF Chat enhancements, Library: Chat commands, and libWrapper enabled on Foundry 0.7.10, on a brand new journal entry.

**Comment by flamewave000** [comment]
Ok, so this may be an issue with 0.7.10 compatibility then. I haven't released any updates, or even tested, my mods on that build yet.

**Comment by marcstraube** [comment]
Same problem on 0.8.6 - but I have not tested without minimal module configuration, yet.

**Comment by flamewave000** [comment]
So I've figured out the issue. For some reason, Atropos has declared the HTML Article tag to be completely filtered out. I don't know what reasoning would be around this, but no matter what, the FoundryVTT server in 0.7.10 and 0.8.x filters them out completely. I've adjusted things to use a different tag. I will be releasing this fix for 0.8.6.

**Comment by duckpuppy** [comment]
Does that mean that it won't be released for 0.7.10? I don't plan on going to the 0.8 series any time soon since I'm waiting on plugins to catch up, so I'm just curious as to whether I need to remove this plugin for now or not.

**Comment by flamewave000** [comment]
Unfortunately I only have so much free time to commit to module work. I won't be doing an entire release for 0.7.10. If you use the module specifically for the Adventure Log, I can make a quick patch for it that you can use. The other features are all working well though.
